src/states/bootstrapping_peer.rs

```diff
@@ -363,7 +371,7 @@ impl Base for BootstrappingPeer {
                 Ok(Transition::Stay)
             }
             _ => {
-                debug!("{} - Unhandled direct message: {:?}", self, msg);
+                trace!("{} - Unhandled direct message: {:?}", self, msg);
```

Comment:
Why this change: Are we getting a lot of messages because other nodes still think we are in previous section?
Is that going to cause problem, (i.e that is why I added the backlog from joining peer etc, because otherwise messages were being dropped).

Reply:
Didn't think this message was important enough to be more than trace. Don't mind changing it back if you disagree.

Reply:
I'm ok either way, i tend to debug with `trace` anyway because otherwise I always got something missing. For me this is more important than `we got that message`, because it can be the root cause of why a message does not reach its destination. So i tend to treat these more carefully.

I was more wondering if there was a change with this PR that made this message appear a lot, and if so if that was OK.